app/src/main/java/org/eyeseetea/malariacare/views/YearPicker.java

```diff
@@ -0,0 +1,72 @@
+package org.eyeseetea.malariacare.views;
+
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.NumberPicker;
+
+import org.eyeseetea.malariacare.R;
+import org.joda.time.DateTime;
+
+
+public class YearPicker extends DialogFragment {
```

Comment:
@manuelplazaspalacio YearPicker should be a EyeSeeTea SDK control and with customizable font as other controls that exists in SDK